internal/wal_push_handler.go

```
@@ -50,7 +50,7 @@ func HandleWALPush(uploader *WalUploader, walFilePath string) {
 	bgUploader.Start()
 	err = uploadWALFile(uploader, walFilePath, bgUploader.preventWalOverwrite)
 	tracelog.ErrorLogger.FatalOnError(err)
-	uploadBulkMetadata(uploader, walFilePath)
+	_ = uploadBulkMetadata(uploader, filepath.Join(getRelativeArchiveDataFolderPath(), filepath.Base(walFilePath)))
```

Why ignore the error here?